docs/templates/template-def-v1.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: Version 1
+permalink: /template-version-1/
+parent: Templates
+---
+
+# Template Definition Version 1 <sup>depricated</sup>
+
+The following objects can be used in template files.
+
+### **project**
+
+General information about the project are stored.
+
+
+| Name        | Description     |      Example  |
```

Comment:
Data Type column is missing

Reply:
it's deprecated - I just want to let it as it was before